### ISSUE TITLE
Billboard sizeInMeters not accurate in Columbus View with WebMercatorProjection

### ISSUE THREAD

**Issue by andreborud** [AnalyticalGraphicsInc/cesium#8026]
I just noticed if you set a Billboard with sizeInMeters to true in Columbus View with WebMercatorProjection the size in meters is not accurate. 

Bellow are screenshots from the sandcastle of AGIs parking lot where each space is about 6m x 3m. The image from the columbus view is almost 1m shorter. 

Not sure if this is the expected behaviour with WebMercatorProjection?

![Screenshot 2019-07-26 11 24 37](https://user-images.githubusercontent.com/5594437/61942364-8ce22a00-af99-11e9-9b3e-a858b86f34d1.png)
Sandcastle example SCENE3D: https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/#c=rVNha9swEP0rwl/igCcn9Zp0qRsKWWGDboFt9JOhKPbZPipLRpKTpaP/fee4bZw2rDCqD+b09O7p6e68FoatETZg2AVTsGELsNhU/GaH+YN0t11o5QQqMIPheaIS1WVwUA4dguUiy/w/iWK0am0J04rNnqQWwjiKhIp4bnT1GQoDYP0P01N++mk6nQbs44iPorOzaBh0GiuUcqWFydjsUbVdWIkCSHbAebiLbdjdcHutC32r12Ck2PJaFYMu6SFRD+/tdzKJng2fjCdHHLP/sxywMGQZ5KKRbsYaRSEVPNtr2VJvSMqZBvrcPaHG3yCXeW7BEa/Xy+cHnfijgI2Gh1d12F4GtvBPkcinx1PK42d43EypDd63UyOXBgvslffLixO+uPr+6+rHcRkqkMP0tcjNAX5Egpg7rFc/ore9GLfG+0QC9qwNZq4k1uTNfpSARdkWKXq7dXgPX9U3cGBs18HDAfUCL7ZuK2He4ZdY1do41hjp0+Q4qGopqPrhqknvwPHU2jYtDp+S4gzXDLOLxHvxvyYeS6Wwlk7yRsqfZCTx5nFI/IM0qUWGqlh2A9lSyvH8ugM553FI29dZTmu5Eqan+Bc

![Screenshot 2019-07-26 11 32 25](https://user-images.githubusercontent.com/5594437/61942365-8d7ac080-af99-11e9-8e56-975634987dcc.png)
Sandcastle example COLUMBUS_VIEW: 
https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/#c=rVRhb9owEP0rJ74QpMyBZoWOUTSNIa0SjGms9EskZJIj8ebYke1AadX/PocUSFq0StP8IbKf3717vjtlQxVsGG5RwTUI3MIINctTsthjTjPcH0dSGMoEqqYLj6BDFDiVEfYP7PkBIaPZ5Hb6+Xa+XNyM71xIafZdyV8YGiZFv5rgDldTVCE1Up0YTgueWh8DEYjSE0FhmGGoCY0i5zEQYFcmNSvIcEw/osrYHRU+WSuZfsFYIWrnXe+SXH7o9XouvG+Ttn915bfcUmPFOF9JqiLoP6sWi6U0RivbJMTb77VXZlhOZCyXcoOK0x3JRNwsg54C8d/9drv+0fBFp3vGMfybZRc8DyJc05ybPuTCbm1Lo5OWTuTWShmVY5V7ImTsHvlsvdZooNbM44MunLYL7VY9VYmdZHCHfxXxHft4G/L8aZ03k0jFHoq55DPFYlYp79cXN2Q0/vZz/OO8jC2QYeFrkUUNPyNhmXusUj9LL3rRKYxXiRY4sbYsMolldd/sR4IsTooi+W+3jj3gjZiiQaXLDtYHtOE2BtrsOA5L/BNLM6kM5Io7dnIMphmntvreKg9/oyGh1kXYwDsEDSK2ARZdB40Xf4SgASGnWtubdc753BoJGsOBZ/m1MC5pxEQ8KweyoCSd4aQECSEDzx5fRxkp+YqqiuIf

Browser: Any

Operating System: Any

**Comment by OmarShehata** [comment]
I was a bit confused at first because the Sandcastles don't zoom into the right billboard. Here's the two versions going to the billboard in the parking lot:

[3D view](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/#c=hVNta9swEP4rwl9igycn9ZpunRsGWWGDboG19JOhKPbZPiZLRpKTJaX/fec4bZw2rDIY6bnnHt2bVsKwFcIaDLtiCtZsDhbbmt/vMH+U7Y5zrZxABWYUfElV9/U+fKt1faf9/QmUQ4dguchz/zFVjFajLWFasctn6bkwjnZCxbwwuv4GpQGw/oeLc37+eTqNL0L2cczH8aezyTQIe5UlSrnUwuQksxfuFtaiBIJGnEe7vY36Sx5udKkf9AqMFBveqHIUsihiORSile6StYq2lFB+0LKVXpOUMy0MuQdCg39BLorCgiPeoFYvCZ3545CNg+OreuwgAxv4r0jsU/Lksv8Fp4OptMFt1xW5MFjioLzfX1n4/PrX3fXv0zJUIIfZW5H7I/yEBDF32KB+RO96MekCHxIJOLDWmLuKWNN3+1EBllVXpPj91uEWfqif4MDYvoO97SlVT0E3sV7oJdZtJMx6w1esG20ca430aXQc1I0UVP5o2WZ/wPHM2s4tiZ6dkhxXDPOr1Hv1IFKPZVJYS5ailfKWIkm9WRIR/8hNapGjKhf9RHaUajK76UHOeRLR8a2X01ouhRko/gM)
[Columbus view](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/#c=hVNhb9owEP0rFl9IpMyBstKNpWgaQ1olGNNo6ZdIyCRH4s2xI9uBQsV/34VACS1a/SHyvXv3fH7nrJgmKw5r0OSWSFiTARheZHS2x5xmtA8HSlrGJeimR56JiUDCWMXQO7KnR4QOJqOH8beH6Xx2N3z0SMbyX1r9gchyJXv1Ax5hMQYdMav0ieG4ZOd+CWUoq57oVqnsXjmHCKTlloOhLI6d51ASXLkyvCwlL80MmLa4Y7JDl1pl3yHRAMb5cHNNrz93u50bj3xs0Vbn01W763qVyoILsVBMxyhzEC4Xz1gCCDUp9fd741eHzEcqUXO1Ai3YhuYyQWN8n8SwZIWwPVJI3KJh8UnLpGqNUlYXUOeeCDl/AjFZLg1YcmbVy4WunJZHWu75URV2koEN/Fek4+DlseTwcS83kyrNt+XUxUTzhNfs/fEqQwfDn/fD35dl0CDLo7ciszP8ggQy91jNP6SXs2iXjdeJCJxYax7bFFndd+eRAk/S0qTO+6PjW7iTY7CgTTXBKrcL5c4t32vDawTGbgT0q8RXnuVKW1Jo4eDTsZDlgqH9/qKI/oKlkTFlWeAfi4KYrwiPb8PGqx8ubJBIMGMwsyyEmGInYaMf+Mg/KxOKxVwmk+pFlpS03R9VIKU08DF8W2WVEguma4r/AA)

This may be related to https://github.com/AnalyticalGraphicsInc/cesium/issues/7985. @likangning93 do you know if this is expected? Would the billboard's size in meters be more or less off relative to the imagery depending on how close you are to the poles?

**Comment by likangning93** [comment]
I don't think this is related to #7985, since that's a vector tiles thing. I think the problem is that the billboard is the correct size in meters, but 2D projections naturally distort distances and/or angles, especially when they cover very large areas. Web Mercator in particular attempts to preserve angles while allowing distances to become more distorted at latitudes further from the equator. In other words, the billboard is fine, it's the car in the imagery that isn't the right size anymore.

**Comment by andreborud** [comment]
I'm not so sure its the car its the car imagery. 

I'm using this function to determine how many meters there are per pixel to have a little scale at the bottom of the map. 

```
function () {
	let width = this.viewer.scene.canvas.clientWidth,
		height = this.viewer.scene.canvas.clientHeight,
		left = this.viewer.scene.camera.pickEllipsoid(new Cartesian2(0, height / 2), Ellipsoid.WGS84),
		right = this.viewer.scene.camera.pickEllipsoid(new Cartesian2(width, height / 2), Ellipsoid.WGS84)

	if (!JSUtil.hasValue(left) || !JSUtil.hasValue(right) || (this.lastScalePosLeft === left && this.lastScalePosRight === right)) {
		return
	}

	this.lastScalePosLeft = left
	this.lastScalePosRight = right

	let d = Cesium.Cartesian3.distance(left, right)
	let nd = d / width

	return nd
}
```

This doesn't take the curvature of earth into account, but when it comes to a few meters it should be accurate enough. 
Anyway, the line I drew based on the function above always seem to be accurate, and the billboard image was always too small when set to size in meters in CV & WebMercatorProjection.

I haven't tested it, but I think you are right that the billboard will be scaled correctly at the equator.
